assets/javascripts/discourse/templates/components/tdc-sign-up.hbs

```diff
@@ -0,0 +1,56 @@
+{{#each site.group_names as |group|}}
+{{#if (is-collective group.full_name)}}
+<div class="user-field collective-selectors">
+  <div class="controls">
+    <label for="{{ group.name }}">
+    {{input type="checkbox" name="debt-type" id=group.name change=(action "collectiveSelected")}}
+    {{group.full_name}}
+    </label>
+  </div>
+</div>
+{{/if}} {{/each}}
+
+{{#if isDebtor }}
+<div class="user-field text">
+    <label for="debt-amount" class="control-label">Debt Amount</label>
+  <div class="controls">
+    {{input value=store.customFields.debt_amount id="debt-amount" name="debt-amount" required=isDebtor}}
+ <!--   &nbsp;{{input-tip validation=debtAmountValidation id="debt-amount-validation"}} -->
+  </div>
+</div>
+{{/if}}
+
+{{#if isInSolidarity }}
+<tr class="input">
+  <td class="label">
+    <label for="solidarity-how-can-you-help">How can you help the debt collective?</label>
```

Comment:
Let's get this merged as soon as you can, but if you could create a ticket for @zakgreene to come up with the right copy for these questions that would be sweet 👌

Reply:
https://github.com/debtcollective/parent/issues/133